hail/python/hail/expr/expressions/typed_expressions.py

```diff
@@ -802,6 +802,13 @@ def __rpow__(self, other):
         return self._bin_op_numeric_reverse('**', other, lambda _: tfloat64)
 
 
+class NDArrayExpression(CollectionExpression):
+
+    def __getitem__(self, item):
+        idxs = to_expr(item, ir.tarray(ir.tint64))
```

Comment:
Why is the type of get item an array of long and not the element type?

Reply:
Oh I see, it's multiple indices.

Reply:
Ya that’s a convention of `__getitem__` that I’m not crazy about